build.gradle

```diff
@@ -108,7 +108,9 @@ subprojects {
         testImplementation 'com.github.stefanbirkner:system-rules'
         testImplementation 'com.squareup.okhttp3:mockwebserver'
         testImplementation 'org.hibernate.validator:hibernate-validator'
-        testImplementation 'org.springframework.boot:spring-boot-starter-test' // MockHttpServletRequest
+        testImplementation('org.springframework.boot:spring-boot-starter-test') { // MockHttpServletRequest
```

Comment:
I think you can use `dependencyManagement` to define exclusion rule in this project.
This can remove copy&paste exclude rule.